react/features/invite/components/index.js

```diff
@@ -2,5 +2,5 @@
 
 export * from './add-people-dialog';
 export * from './dial-in-summary';
-export * from './info-dialog';
 export * from './callee-info';
+export * from './security-dialog';
```

Comment:
The security dialog is not related with inviting people, is it? Then let's move it some other feature.